sql-plugin/src/main/scala/com/nvidia/spark/Utils.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark
+
+import java.util.concurrent.TimeUnit.NANOSECONDS
+
+/** Utilities derived from Apache Spark's Utils class */
+object Utils {
```

Comment:
I was just following the Spark precedent here.  I almost added a Trampoline function for it, but it was so small I decided it was less risk to have our own version in case they changed/moved it in some Spark release.

I'll update the name.